modules/search-api/src/main/scala/io/renku/search/api/data/SearchEntity.scala

```diff
@@ -71,13 +83,14 @@ object SearchEntity:
   given Schema[SearchEntity] = {
     val derived = Schema.derived[SearchEntity]
     derived.schemaType match {
-      case s: SchemaType.SCoproduct[_] =>
+      case s: SCoproduct[_] =>
         derived.copy(schemaType =
           s.addDiscriminatorField(
             FieldName(discriminatorField),
             Schema.string,
             List(
-              implicitly[Schema[Project]].name.map(SchemaType.SRef(_)).map("Project" -> _)
+              implicitly[Schema[Project]].name.map(SRef(_)).map("Project" -> _),
+              implicitly[Schema[User]].name.map(SRef(_)).map("User" -> _)
```

Comment:
In scala3 there is now `summon[…]` - not quite sure what the difference is, though